apollo-router/src/plugins/telemetry/mod.rs

```diff
@@ -77,17 +92,19 @@ use crate::SupergraphRequest;
 use crate::SupergraphResponse;
 
 pub(crate) mod apollo;
+pub(crate) mod apollo_exporter;
 pub(crate) mod config;
 mod metrics;
 mod otlp;
 mod tracing;
 
-static SUPERGRAPH_SPAN_NAME: &str = "supergraph";
+static REQUEST_SPAN_NAME: &str = "request";
```

Comment:
Also keep the `SUPERGRAPH_SPAN_NAME` for the exporter

Reply:
Fixed in [f7d8aa5](https://github.com/apollographql/router/pull/1514/commits/f7d8aa5481f58454542390e87e0f2cfa4426cc38)